swagger-doc-extractor/README.md

```diff
@@ -15,5 +17,5 @@ however we don't want the overhead of starting up an entire Alfred API stack.
 
 ## Usage
 ```bash
-./gradle --quiet :swagger-doc-extractor:run
+./gradlew --quiet :swagger-doc-extractor:run > ./swagger-doc-extractor/build/apix-swagger.json
```

Comment:
... or "swagger.json" I assume? This specific output example filename seems to imply some mechanism?

Reply:
Generic names are never a good thing imo. if you're going through editor tabs or os-windows Its better to have some description. Im tempted to add a timestamp to it as well, but that would make the command uglier.

Reply:
Yeah, but 'build' dirs are used by every build tool, and implies it can be written by them.
Anyway, this is not a blocker for me.